vars/mavenTemplate.groovy

```diff
@@ -69,7 +69,7 @@ def call(Map parameters = [:], body) {
                                     workingDir: '/home/jenkins/',
                                     //resourceLimitMemory: '640Mi',
                                     envVars: [
-                                            //envVar(key: '_JAVA_OPTIONS', value: '-Duser.home=/root/ -XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -Dsun.zip.disableMemoryMapping=true -XX:+UseParallelGC -XX:MinHeapFreeRatio=5 -XX:MaxHeapFreeRatio=10 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Xms10m -Xmx192m'),
+                                            envVar(key: '_JAVA_OPTIONS', value: '-Duser.home=/root/ -XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -Dsun.zip.disableMemoryMapping=true -XX:+UseParallelGC -XX:MinHeapFreeRatio=5 -XX:MaxHeapFreeRatio=10 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Xms10m -Xmx192m'),
```

Comment:
@rupalibehera @rawlingsj I think model need lot of memory with recent upgrade changes>
It would be better if make -Xmx512m at least.

Reply:
@jerboaa is the person to comment on this.  @jerboaa we noticed that we were getting failures on our kubernetes cluster now that we have the new base build pods.  The original podTemplate only updated the openshift pods but I wondered if we now need the JVM OPTS when running on kubernetes as well?  We're gonna try this in the meantime to see what happens.

Reply:
@hrishin @rawlingsj Those options were tuned to fit the OSIO booster needs, specifically vertx, spring-boot and wfswarm. `-Xmx` option and the container memory limit are tightly coupled. The goal there was to keep memory consumption minimal, while still able to build boosters. Not sure what the kubernetes use case is. From what I see it does not have any memory limit restrictions on the container side. Perhaps it makes sense to remove the `-Xmx` option altogether.

Without understanding the problem better it's hard to tell though. I'm not sure when the "Docker socket" branch is taken and whether or not different workloads run on those.

Reply:
@jerboaa Thanks a lot for the advice and makes sense. @rawlingsj @rupalibehera  let us identify the root cause and reproduce the issue.
Then will try to send PR if required. :)